lms/static/sass/base/_variables.scss

```diff
@@ -77,11 +77,21 @@ $gray-l4: tint($gray,80%);  // #e5e5e5
 $gray-l5: tint($gray,90%);  // #f2f2f2
```

Comment:
Hmmm, my suggestion may not have been clear, but please use existing Sass color variables if possible. 

1. When there is not a comparable (near similar color) value, please do not add more variables into this file. 
2. Any specific colors you need should just be hardcoded into your specific Sass following our color use conventions (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#colors)